pandas/tests/groupby/test_missing.py

```diff
@@ -43,6 +43,17 @@ def test_ffill_missing_arguments():
         df.groupby("b").fillna()
 
 
+def test_ffill_with_string_column():
+    # GH 40250
+    result = (
+        DataFrame({"a": pd.array([None, "a"], dtype="string"), "b": [0, 0]})
```

Comment:
can you change the example so it *also* fills a forward value (e.g. make the input '[None, "a", None]'